src/editors/utils/formatLibraryImgRequest.ts

```diff
@@ -0,0 +1,134 @@
+import StrictDict from './StrictDict';
+
+/**
+ * A dictionary that maps file extensions to their corresponding MIME types for images.
+ *
+ * @example
+ * acceptedImgMimeTypes.gif // "image/gif"
+ */
+
+const acceptedImgMimeTypes = StrictDict({
+  gif: 'image/gif',
+  jpg: 'image/jpg',
+  jpeg: 'image/jpeg',
+  png: 'image/png',
+  tif: 'image/tiff',
+  tiff: 'image/tiff',
+  ico: 'image/x-icon',
```

Comment:
Is there any way we can avoid hard-coding this list of image mimetypes? It doesn't seem like something the frontend should be concerned with. Or is it required for `FILES_AND_UPLOAD_TYPE_FILTERS` ?

Also, I have been using `.avif` image files in the authoring MFE with no problems, so that should be added to the lists. But it's also an example of why it's better to avoid such lists if at all possible.

![Screenshot](https://github.com/user-attachments/assets/41a8a765-7b75-49f6-acc8-406d5cadea05)

Reply:
If we have to list them out explicitly, webp and svg should be there as well.

Reply:
I can remove the list of mimetypes, however it is important to check have a filter. 

I take those ones from the input. I can use that instead of create a new one 

https://github.com/openedx/frontend-app-authoring/blob/d99e3f0f62f661381f9b257bd7fe98612f09c520/src/editors/sharedComponents/ImageUploadModal/SelectImageModal/utils.js#L39

Reply:
>  it is important to check have a filter.

Sure, but can that check be `image/*`? That's a much better check than maintaining a list of filetypes.

Reply:
Also, maybe you can [get the browser to provide the mime type](https://stackoverflow.com/a/29672957) ?

Reply:
I was thinking to check just the file extension with  `Object(acceptedImgKeys).keys()` and just check the file extension against that array, in that way we centralize the information, I noticed that in fact the `GalleryCard` is not using the value and `when the component is added in a course `FILES_AND_UPLOAD_TYPE_FILTERS`is not affected, so I can skip it for libreries. What do you think?

Reply:
Great! If we don't need it, please just skip it.